## TASK

It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

Set up the hypothesis test.
Write down all the steps followed for setting up the test.
Calculate the test statistic by hand and also code it in Python. It should be 4.76190. What decision can you make based on this calculated value?

**Null Hypothesis**

H0: μ = 120 
    
**Alternative Hypotheis**

HA: μ ≠ 120

Two Tailed Test


### Sample

**Sample Size** n = 100

**Average mean** x̄ = 130.1  (mm Hg)

**Standard Deviation** S = 21.21 (mm Hg)

**Mean of population** is unknown, therefore **t-distribution** is used with degree of freedom of n-1.

***

t = (x̄ - μ) / (S / n^0.5) = (130.1 - 120) / (21.21 / 100^0.5) = 4.76190

In [1]:
xbar = 130.1
mu = 120
S = 21.21 
n = 100

t = (xbar - mu) / (S / n**0.5)

print(round(t, 4))

4.7619


### Hypothesis Test

In [2]:
from scipy import stats
tc = +-stats.t.ppf(1-(0.05/2),df=99)
print(round(tc, 4))

-1.9842


With α = 0.05, t should be within -t(0.0025,99) ≤ t ≤ +t(0.0025,99) → -1.9842 ≤ 4.7619 ≤ 1.9842.<br>
This formula is not true, therefore H0 is **rejected**, the mean systolic blood pressure is **not** μ = 120 mm Hg.